.github/actions/setup/action.yaml

```diff
@@ -30,6 +30,11 @@ runs:
       shell: bash
       run: curl -o /usr/local/bin/uds -L https://github.com/defenseunicorns/uds-cli/releases/download/v${{ inputs.udsCliVersion }}/uds-cli_v${{ inputs.udsCliVersion }}_Linux_${{steps.setUdsCliArch.outputs.ARCH}} && chmod +x /usr/local/bin/uds
 
+    - name: Install Lula
+      uses: defenseunicorns/lula-action/setup@095636b7880051e11b05f10a582fdd911526161c
+      with:
+        version: v0.3.0
+
```

Comment:
The Lula linting should also be run in this repository's lint Github workflow against a file in the root of this repo.

Reply:
Added it to the lint github action